Mods.md

```diff
@@ -0,0 +1,67 @@
+This guide will describe how mods and their triggers attach to FullScreenShenanigans projects.
+
+### Table of Contents
+1. [ModAttachr](#modattachr)
+    1. [Adding Mods](#adding-mods)
+    2. [Event Hooks](#event-hooks)
+
+# ModAttachr
+
+ModAttachr is a module that allows for extensible triggered mod events.
+
+A mod is a custom modification that changes how the game normally runs and/or how the player interacts with the game.
+Each mod can define any number of events, keyed by string, to call when they are fired.
+When an event is fired in the game, ModAttachr calls the corresponding events of all enabled mods that have a defined event under that name.
+
+## Adding Mods
+
+A mod can be added with `addMod`.
+
+```typescript
+ModAttachr.addMod({
```

Comment:
Just noticed many of these samples are missing the conventional `e` in their names. `ModAttachr` vs `ModAttacher`.